.github/workflows/sonar-test-pr.yml

```diff
@@ -0,0 +1,44 @@
+name: Maven Pull Request Sonar Test
+
+on:
+  workflow_run:
+    workflows: [Maven Pull Request Build]
+    types:
+      - completed
+
+jobs:
+  on-success:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.head_repository.fork == true }}
+    steps:
+      - name: Fetch PR number
+        id: fetch
+        uses: chrweiss/pull-request-number-action@v0.2
```

Comment:
Is this correct using an action belonging to your personal account? Could we add it under a FA³ST id?

Reply:
The action is not at all AAS related, so publishing it under as part of FA³ST would be misleading.
I'd just consider it as an artbitrary github action/OSS tool used in the workflow.
I'm not aware of other tools providing the information I am scraping with this tool. However, it is possible that alternatives do exist.

Reply:
I also dislike the idea of having a dependecy on your personal repository. I would say the best place to host this to create a repository under FraunhoferIOSB organization. @chrweiss  feel free to contact me if you do now have sufficient rights to create a new project.